Kinvey.Net/Kinvey.Net.csproj

```diff
@@ -76,10 +76,6 @@
       <Project>{444456B1-5B55-48D6-9637-00ADCA92B7C9}</Project>
       <Name>Kinvey-Utils</Name>
     </ProjectReference>
-    <ProjectReference Include="..\RestSharp.Portable\RestSharp.Portable.csproj">

```

Comment:
Can we remove this yet, since this is a project that is currently supported, and not the new .NETStandard-based project?

Reply:
the `Kinvey` and `Kinvey.Core` shared the same code base, so no need to keep `RestSharp`